### PR TITLE
Remove redundant record from readme

### PR DESCRIPTION
**Pull request by DangerDC**
Removed record from same pilot and frequency, with shorter distance and more power.

### CODE REVIEW

**Comment by nkruzan** [comment]
Maybe leave that one in and remove the 10km on next line instead

**Comment by JyeSmith** [comment]
@wvarty please review

File: README.md [hunk 1]
```diff
@@ -110,7 +110,6 @@ Anyone can add an entry to the table, and entries should include the:
 | ---- | -------- | -------- | --------- | ---------------------- | ------------ | ----------- |
 | 40Km | 900M | 50HZ | 10mW | No | Snipes | https://www.youtube.com/watch?v=0QWN9qWoSYY |
 | 33Km | 2.4G | 250HZ | 100mW | No | Snipes | https://www.youtube.com/watch?v=GkOCT17a-DE |
-| 30Km | 900M | 50HZ | 1W | No | Snipes | https://www.youtube.com/watch?v=SbWvFIpVkto |
```
Comment (wvarty) [inline README.md:113]: Might as well remove the entry below as well? (10km 2.4 250HZ).

Comment (maybenikhil) [reply README.md:113]: agreed

**Comment by JyeSmith** [comment]
Changes included in https://github.com/ExpressLRS/ExpressLRS/pull/711

**Pull request closed without merging**